lib/guest/linux.go

```diff
@@ -27,7 +27,7 @@ import (
 )
 
 const (
-	linuxGuestID = "other3xLinux64Guest"
+	linuxAlternateGuestName = "vSphere Integrated Containers - Container 0.9.0"
```

Comment:
Whoops, I did not mean to include this test code.  Though, it's a really simple implementation.  I might as well just include the code and tests with this PR.